crates/router/src/core/payments/transformers.rs

```diff
@@ -98,6 +98,13 @@ where
 
     let customer_id = customer.to_owned().map(|customer| customer.customer_id);
 
+    let is_connector_new_version = state
```

Comment:
in env add a variable called multiple_api_version_supported_connectors and use that to fetch this values and the connector_api_version becomes optional.